### PR TITLE
Add new requirements into requirements.txt

### PR DESCRIPTION
**Pull request by zachmullen**
@cjh1 this is the commit I forgot to push on #1795

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,6 +3,8 @@ bcrypt==2.0.0
 boto==2.40.0
 cffi==1.8.3
 CherryPy==6.0.1
+functools32==3.2.3.post2
```
Comment (cjh1) [inline requirements.txt]: Whats this needed for?

Comment (zachmullen) [inline requirements.txt]: Transitive dependency of jsonschema. This file is the dump of a `pip freeze` in a known working state.

Comment (cjh1) [inline requirements.txt]: OK, that is how we keep this up to date?

Comment (zachmullen) [inline requirements.txt]: This file is not actually used in CI or during pip install, the authoritative listing is in setup.py. @jbeezley can better explain the purpose of requirements.txt.

Comment (jbeezley) [inline requirements.txt]: I'm planning to do a D&A lightning talk about it some time.

**Pull request merged**